src/utils/daysjs.ts

```diff
@@ -0,0 +1,8 @@
+import dayjs, { ConfigType } from "dayjs"
+import relativeTime from "dayjs/plugin/relativeTime"
+
+dayjs.extend(relativeTime)
+
+export const relTime = (date: ConfigType) => {
```

Comment:
```suggestion
export const relTime = (date: ConfigType) => dayjs().to(dayjs(date)
```

util化ナイスです！！
関数の中身が`return`だけの場合`{}`を省略することもできます！
ご参考までに🙇